<launch>
  <node name="object_state_publisher" pkg="knowrob_objects" type="object_state_publisher.py" output="screen">
    <param name="hz" value="1" />
  </node>
</launch>
